docs/changes/io.ascii/16005.bugfix.rst

```diff
@@ -0,0 +1,5 @@
+The ``io.ascii`` Python and C table readers were updated to use a 64-bit integer field by
+default when reading a column of integer numeric data. This changes the default behavior
+on Windows and potentially 32-bit architectures. Previously on those platforms, table
+columns with any long  integers which overflowed the 32-bit integer would be returned
+as a string column. The new default behavior is consistent with `numpy` v2 and `pandas`.
```

Comment:
We do not do "live linking" in change log because we do not want to go back in time to fix broken links from the past.

```suggestion
as a string column. The new default behavior is consistent with ``numpy`` v2 and ``pandas``.
```

Also, do you want to mention IPAC here?

Reply:
See the latest revison.